R/plot-construction.r

```diff
@@ -60,46 +60,88 @@ add_ggplot <- function(p, object, objectname) {
   if (is.null(object)) return(p)
```

Comment:
This can now be a method of the generic

Reply:
Oh - Didn’t knew you could dispatch on `NULL`